.gitattributes

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2024 Justin Myers for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
+
+* text eol=lf
```

Comment:
Is the `.gitattributes` for helping with Windows?  I haven't had an issue with `ruff` and EOLs.

Reply:
It is for helping with windows, not specifically related to `ruff` but was discussed in the same larger conversation during the meeting.

Though my understanding is this was a fix for something that pylint complained about, perhaps with ruff replacing pylint we may not need this change any more?

Reply:
This will make sure they commit fine, even if they forget to run pre-commit, allowing small changes to still pass CI